### ISSUE TITLE
tweepy.error.TweepError: [{'code': 385, 'message': 'You attempted to reply to a Tweet that is deleted or not visible to you.'}]

### ISSUE THREAD

**Issue by FujiMakoto** [FujiMakoto/twitter-saucenao#11]
We should cleanly handle errors where people delete their posts mentioning the bot before we can reply.

```
Traceback (most recent call last):
  File "/home/saucenao/twitter-saucenao/twsaucenao/server.py", line 106, in check_mentions
    self.send_reply(tweet, sauce, blocked=tweet_parser.blocked)
  File "/home/saucenao/twitter-saucenao/twsaucenao/server.py", line 381, in send_reply
    raise error
  File "/home/saucenao/twitter-saucenao/twsaucenao/server.py", line 374, in send_reply
    comment = api.update_status(reply, in_reply_to_status_id=tweet.id, auto_populate_reply_metadata=True)
  File "/home/saucenao/twitter-saucenao/venv/lib/python3.7/site-packages/tweepy/api.py", line 193, in update_status
    )(post_data=post_data, *args, **kwargs)
  File "/home/saucenao/twitter-saucenao/venv/lib/python3.7/site-packages/tweepy/binder.py", line 250, in _call
    return method.execute()
  File "/home/saucenao/twitter-saucenao/venv/lib/python3.7/site-packages/tweepy/binder.py", line 233, in execute
    raise TweepError(error_msg, resp, api_code=api_error_code)
tweepy.error.TweepError: [{'code': 385, 'message': 'You attempted to reply to a Tweet that is deleted or not visible to you.'}]
```

It's very rare, but it happens.

**Comment by CandyKan3** [comment]
Hey, I would love to begin contributing to this project. Is there a discord server to discuss issues/contributions to this project or get assigned to certain issues?

**Comment by FujiMakoto** [comment]
@CandyKan3 Hey there! I don't have an official guide for contributing yet but I'd be more than happy to accept help if you're interested in contributing!

I have a Discord for the project which you're welcome to join here:

https://discord.gg/p2VQTFP

**Comment by FujiMakoto** [comment]
We addressed this in #17 and now properly handle all Tweepy errors in a single method,
https://github.com/FujiMakoto/twitter-saucenao/blob/master/twsaucenao/server.py#L283-L306
